src/MICore/Debugger.cs

```diff
@@ -712,7 +712,7 @@ public Task CmdBreakInternal()
                 {
                     if (_debuggeePids.Count > 0)
                     {
-                        debuggeePid = _debuggeePids.First().Value;
+                        debuggeePid = _transport.DebuggerPid; // _debuggeePids.First().Value;
```

Comment:
Do we still need this comment?

Reply:
Sorry, this was a customer investigation that i was doing. Customer says we should be sending signal to GDB and not the debuggee and that would pause the debuggee. I was making sure it worked. I need to move this into its own PR.

Reply:
I think you forgot to fix this